File: src/main/java/com/banchan/model/entity/UserAuthInfo.java
//package com.banchan.model.entity;
//
//import lombok.Data;
//
//import javax.persistence.Column;
//import javax.persistence.Entity;
//import javax.persistence.GeneratedValue;
//import javax.persistence.GenerationType;
//import javax.persistence.Id;
//import java.time.LocalDateTime;
//
//@Data
//@Entity(name = "user_auth_infos")
//public class UserAuthInfo {
//    @Id
//    @GeneratedValue(strategy = GenerationType.IDENTITY)
//    private Long id;
//
//    @Column(name = "user_id")
//    private Long userId;
//
//    @Column(name = "token_key")
//    private String tokenKey;
//
//    @Column(name = "use_yn")
//    private String useYn;
//
//    @Column(name = "updated_at")
//    private LocalDateTime updatedAt;
//
//    @Column(name = "created_at")
//    private LocalDateTime createdAt;
//}

File: src/main/java/com/banchan/model/entity/UserAuthInfo.kt
package com.banchan.model.entity

import lombok.Data
import java.time.LocalDateTime
import javax.persistence.*

@Data
@Entity(name = "user_auth_infos")
data class UserAuthInfo (
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        val id: Long? = null,

        @Column(name = "user_id")
        val userId: Long,

        @Column(name = "token_key")
        val tokenKey: String,

        @Column(name = "use_yn")
        val useYn: String = "Y",

        @Column(name = "updated_at", nullable = true)
        val updatedAt: LocalDateTime? = LocalDateTime.now(),

        @Column(name = "created_at")
        val createdAt: LocalDateTime? = LocalDateTime.now()

)